Automattic/jetpack
# Jetpack: add a transform from/to core/video to VPBlock

This PR adds transforms from/to core/video jetpack/videopress-block.

<img width="919" alt="Screen Shot 2022-07-21 at 3 36 34 PM" src="https://user-images.githubusercontent.com/77539/180298030-d01f3c96-a4c4-49b7-ae22-0962ed9653e3.png">

_Disclaimer: it works considering that the current implementation is active, too. We'll tackle transformations by disabling the current implementation of VideoPress (v5) in follow-up PRs._

Fixes #

#### Changes proposed in this Pull Request:

* Jetpack: add a transform from/to core/video to VPBlock

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to the Block editor
* Create a core/video or a VPBlock
* Upload the video
* confirm the transform works as expected

https://user-images.githubusercontent.com/77539/180297525-5de8bf05-558e-4f3b-af51-fb9b33b0287f.mov

_Disclaimer: it works considering that the current implementation is active, too. We'll tackle transformations by disabling the current implementation of VideoPress (v5) in follow-up PRs._



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1202624432767029
